Sofa/Component/Constraint/Lagrangian/Model/src/sofa/component/constraint/lagrangian/model/BilateralInteractionConstraint.inl

```diff
@@ -411,20 +411,11 @@ void BilateralInteractionConstraint<DataTypes>::handleEvent(Event *event)
         const KeypressedEvent *ev = static_cast<KeypressedEvent *>(event);
         switch(ev->getKey())
         {
-
         case 'A':
         case 'a':
-            if (d_activate.getValue())
-            {
-                msg_info() << "Unactivating constraint";
-                d_activate.setValue(false);
-            }
-            else
-            {
-                msg_info() << "Activating constraint";
-                d_activate.setValue(true);
-            }
-            
+            msg_deprecated() << "This component was implemented with hard coded user interactions. This behavior was deprecated in PR#4163. "
+                                "If you need the old behavior, you need to re-implement with a controller. This can be done"
+                                " c++ or python. If you see this message  and are unable implement a controller on your own, please report at https://github.com/sofa-framework/sofa/pull/4163";
```

Comment:
```suggestion
                                "If you need the old behavior, you need to re-implement with a controller. This can be done in"
                                " c++ or python. If you see this message  and are unable implement a controller on your own, please report at https://github.com/sofa-framework/sofa/pull/4163";
```